Sprint-1/exercises/count.js

```diff
@@ -4,3 +4,5 @@ count = count + 1;
 
 // Line 1 is a variable declaration, creating the count variable with an initial value of 0
 // Describe what line 3 is doing, in particular focus on what = is doing
+
+//Line 3 is reassigning count to become ann adition of its previous value plus one.
```

Comment:
Would you try feeding the code to ChatGPT to get a second opinion or to see how else the code can be described?
From time to time you can learn some terminology or more concise way to describe code.

In programming, we normally just describe `count = count + 1` as increment `count` by 1.

Reply:
Thanks for the tip CJ. It makes a lot of sense, I'll use GPT more to understand specific terminology.

I was just a bit hesitant on using AI but having it as a guide works as well.